src/components/data-management/SamplesTable.jsx

```diff
@@ -48,6 +50,18 @@ const exampleDatasets = [
   },
 ];
 
+// If a sample exists in projects but not in the samples file,
+// there is an error uploading the sample. We should show an error for that sample.
+const sampleUploadErrorTemplate = {
+  ...sampleTemplate,
```

Comment:
what else is inside `sampleTemplate`?

Reply:
const sampleTemplate = {
    name: null,
    projectUuid: null,
    uuid: null,
    type: null,
    createdDate: null,
    lastModified: null,
    complete: false,
    error: false,
    fileNames: [],
    files: {},
    metadata: {},
  };